lib/bolt/config/options.rb

```diff
@@ -194,7 +195,7 @@ module Options
           },
           additionalProperties: {
             description: "Configuration for the logfile output.",
-            type: Hash,
+            type: [String, Hash],
```

Comment:
```suggestion
            type: [String, Hash],
            enum: ["disable"],
```

This will restrict string values to only allow `disable` in the schema.

Reply:
Ah perfect, I was looking for something like that but failed to find it.